vendor/src/github.com/docker/engine-api/types/client.go

```diff
@@ -178,6 +178,7 @@ type ImageLoadResponse struct {
 
 // ImagePullOptions holds information to pull images.
 type ImagePullOptions struct {
+	All           bool
```

Comment:
@aaronlehmann true. I'll add a comment on this next PR I do on engine-api :wink:. It should get *in* docker/docker before next release but we shouldn't block this PR for it :angel: